packages/quantic/force-app/main/default/lwc/quanticResultQuickview/quanticResultQuickview.css

```diff
@@ -3,6 +3,10 @@
 .quickview__content-container {
   overflow: auto;
   border: none;
+  display: flex;
+}
+.slds-modal__content {
+  height: 90%;
```

Comment:
Also you can add the styles of this class to the already existing class `quickview__content-container` since both are used in the same element instead of creating a new class.